PyRoute/Inputs/StarlineParser.py

```diff
@@ -0,0 +1,70 @@
+"""
+Created on Dec 27, 2023
+
+@author: CyberiaResurrection
+"""
+
+from lark import Lark
+import re
+
+
+def dashrepl(m):
+    group = m.group().replace('  ', ' ')
+    return group
+
+
+class StarlineParser:
+
+    starline_grammar = r"""
+    starline: position starname trade extensions nobles base zone world_alg residual?
+
+    position: /^([0-3]\d[0-4]\d)/
+
+    starname: /(.{15,}) ([\w\?]{7,7}-[\w\?]){1,1} /
```

Comment:
Isn't `{1,1}` the default for a setup? 
```suggestion
    starname: /(.{15,}) ([\w\?]{7,7}-[\w\?]) /
```

Reply:
That's done, tests are passed, so up she goes.